pkg/utils/constants.go

```diff
@@ -83,7 +87,8 @@ const (
 const (
 	// DefaultNamespace is the Kubernetes namespace that is used by default for
 	// the Velero server and API objects.
-	DefaultNamespace = "velero"
+	DefaultNamespace  = "velero"
+	DefaultSecretName = "cloud-credentials"
```

Comment:
Can we give a more specific name, eg CloudCredentialSecretName. There are several secrets in the velero namespace, so not sure if we should call this default secret.